build_tools/python/e2e_model_tests/test_definitions.py

```diff
@@ -18,7 +18,7 @@
 
 class CMakePlatform(enum.Enum):
   """Enum of CMake system platform string."""
-  ANDROID_ARMV8_A = "arm64-v8a-Android"
+  ANDROID_ARMV8_A = "aarch64-Android"
```

Comment:
I still don't understand what the difference between arm64 and aarch64 is. I think the v8a is maybe relevant though?

Reply:
Probably @antiagainst would know best